config/opal_setup_wrappers.m4

```diff
@@ -130,6 +130,12 @@ AC_DEFUN([OPAL_SETUP_WRAPPER_INIT],[
                   [enable rpath/runpath support in the wrapper compilers (default=yes)])])
     AS_IF([test "$enable_wrapper_rpath" != "no"], [enable_wrapper_rpath=yes])
     AC_MSG_RESULT([$enable_wrapper_rpath])
+
+    AC_ARG_ENABLE([new-dtags],
+                  [AS_HELP_STRING([--enable-new-dtags],
+                  [enable new-dtags in the wrapper compilers if linker supports it (default=yes)])])
```

Comment:
This option is moot if `--disable-wrapper-rpath`, right?  If so, that should be included in the help message.